app/javascript/components/rooms/Presentation.jsx

```diff
@@ -20,16 +19,21 @@ export default function Presentation() {
             <div className="user-icon-circle rounded-circle d-block mx-auto mb-3">
               <FontAwesomeIcon icon={faCloudArrowUp} className="fa-4x text-primary d-block mx-auto pt-3" />
             </div>
-            <Card.Title className="text-primary"> Click to upload or drag and drop </Card.Title>
+            <Card.Title className="text-primary">
+              <label htmlFor="file" className="presentation-upload">Click to Upload
+                <input
+                  id="file"
+                  style={{ display: 'none' }}
```

Comment:
We shouldn't be doing inline styles like this. Check for a bootstrap class that can do this for you